### PR TITLE
Use new table component in repeating groups

### PR DESCRIPTION
**Pull request by bjosttveit**
## Description

Updating repeating groups to use the new `Table`-component from the design system instead of the old `AltinnTable`. In order to achieve a styling similar to that in figma (full width and lining up columns with the rest of the page for example) a lot of custom styling was added.

Note: Will conflict heavily with #714 

Desktop:

image

Nested desktop:

image

Mobile:
image

Nested mobile:
image

## Related Issue(s)
- #580 
- #107 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- ~~Relevant automated test added (if you find this hard, leave it and we'll help out)~~
- [x] All tests run green

## Documentation
- ~~User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)~~

### CODE REVIEW

File: src/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx [hunk 1]
```diff
@@ -124,6 +101,25 @@ export function GroupContainer({ id, container, components }: IGroupProps): JSX.
     [dispatch, id],
   );
 
+  const addButon = () => (
```
Comment (olemartinorg) [inline src/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx:104]: Buton, is that french for something? :sweat_smile: :shrug:

File: src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx [hunk 2]
```diff
@@ -59,7 +59,66 @@ export interface IRepeatingGroupTableProps {
 
 const theme = createTheme(altinnAppTheme);
 
+// Page padding could be moved to css-variables
+const ps = 24;
+const pm = 84;
+const pl = 96;
+const m = 15;
```
Comment (olemartinorg) [inline src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx:66]: I see the reasoning behind keeping these variable names short, but this way they don't make any sense for me. Could you add a comment to describe them at, least? :thinking: 

My interpretation:
- `ps` = padding, small
- `pm` = padding medium
- `pl` = padding large
- `m` = something else that is medium? :shrug:

Comment (olemartinorg) [reply src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx:66]: Also, is this kind-of a re-implementation of `FullWidthWrapper`? Could something from that component be re-used here?

**Review by olemartinorg — APPROVED** [review]
:star_struck:

File: src/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx (hunk 1 above)
Comment (bjosttveit) [reply src/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx:104]: indeed 🥖 🍷 🇫🇷

File: src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx (hunk 2 above)
Comment (bjosttveit) [reply src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx:66]: Never heard of that, will look into it

Comment (bjosttveit) [reply src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx:66]: Yes I guess it kind of is, but it also has styling to line up the cell-content with the page, so there are not that many lines to save by reusing this

Comment (bjosttveit) [reply src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx:66]: Sure, I could use the fullwidthwrapper with a small modification to it so i could add classnames through props

**Review by olemartinorg — APPROVED** [review]
:godmode:

**Pull request merged**